Code/Network/RKPaginator.m

```diff
@@ -230,10 +227,12 @@ - (void)loadPage:(NSUInteger)pageNumber
         return deserializedResponseBody;
     }];
     [self.objectRequestOperation setCompletionBlockWithSuccess:^(RKObjectRequestOperation *operation, RKMappingResult *mappingResult) {
+        [self finish];
```

Comment:
I could make `RKObjectRequestOperation` to make `finished` publish, but I think this solution is cleaner. 

Apple says

    The completion block you provide is executed when the value in the finished property changes to YES. 

So it will always be called, but we avoided race condition when it success was called before isFinished KVO notification.